deployer/run.sh

```diff
@@ -169,9 +169,13 @@ keytool -noprompt -import -v -trustcacerts -alias $hawkular_cassandra_alias -fil
 
 echo "Importing the CA Certificate into the Cassandra Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias cassandraca -file ${dir}/hawkular-cassandra-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
 
 echo "Importing the CA Certificate into the Hawkular Metrics Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricsca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
```

Comment:
Can you explain how using the cluster CA would help in this situation?

I could pass in the CA key as a secret which would allow me to sign my own certificates with the CA. But there is no way to synchronize the serial numbers being used. So the certificates signed within the deployer will contain a serial number as another certificate signed outside of the deployer using the normal signing methods.

Firefox and Chrome block access if it sees two certificates containing the same serial value from a CA. You don't even get an option to by pass it. Firefox at least gives you an error message, while Chrome pretends the site is unavailable.